test-engineering/contract-tests/client/tests/conftest.py

```diff
@@ -6,11 +6,14 @@
 import os
 import pathlib
 from functools import lru_cache
-from typing import Dict
+from typing import Any, Dict, Set
 
 import pytest
-import requests
 import yaml
+from requests import Response as RequestsResponse
+
+from exceptions import MissingKintoDataFilesError
+from kinto import KintoAttachment, KintoEnvironment, KintoRecord, get_record
```

Comment:
We define models for kinto-related objects in both `kinto.py` and `models.py` now. Some are loading `scenarios.yml` and others are for loading response content from kinto.

Something to consider in the future is whether we can make it clearer where models should be defined and how they should be named. From the import statements alone it's a bit confusing that `KintoSuggestion` is imported from `models.py` but `KintoAttachment` for example is imported from `kinto.py`.

Reply:
change applied: https://github.com/mozilla-services/merino/pull/409/commits/466d2c2d7a7804ee8754ac9e4b65ada1856f9974

I've moved the KintoSuggestion into the kinto module and also removed `config.kinto_suggestions` in favour of re-using the `kinto_records` fixture.